app/views/spaces/index.html.erb

```diff
@@ -1,24 +1,21 @@
 <div class="cb-section">
-  <div class="cb-container">
-    <h1>slack.com Integration</h1>
-  </div>
-</div>
-
-<div class="cb-section">
-  <div class="cb-container">
-    <p>This add-on will connect your coworking space on Cobot with your <a href="http://slack.com">slack.com</a> account.</p>
+  <div class="cb-container mw6">
+    <h1>Slack Integration</h1>
 
-    <p>New members on Cobot will automatically be invited to join your Slack team, so you don't have to add them manually.</p>
+    <p>This add-on will connect your space on Cobot with your <a href="http://slack.com">slack.com</a> account. New members on Cobot will automatically be invited to join your Slack team, so you don’t have to add them manually.</p>
```

Comment:
Much nicer copy and thanks for catching the correct apostrophe!

Reply:
🙂